safe/gui/tools/help_dialog.py

```diff
@@ -33,7 +33,9 @@ def __init__(self, parent=None, message=None):
         :type parent: QWidget
         """
 
-        QtGui.QDialog.__init__(self, parent)
+        QtGui.QDialog.__init__(
+            self, parent,
+            flags=Qt.WindowMinimizeButtonHint | Qt.WindowMaximizeButtonHint)
         self.setupUi(self)
         self.setWindowTitle(self.tr('InaSAFE %s Help' % get_version()))
```

Comment:
Yes better than `Dialog` but I though you wanted to rename your dialog by removing the inasafe version, like we did in the minimum needs calculator